pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -165,25 +165,42 @@ func (cp *capacityPlugin) OnSessionOpen(ssn *framework.Session) {
 		return !overused
 	})
 
-	ssn.AddAllocatableFn(cp.Name(), func(queue *api.QueueInfo, candidate *api.TaskInfo) bool {
```

Comment:
We'd better explain why the allocatable is no needed and replaced by preemptive, the overall explaination is that in allocate action we need check the ancestor queues' capability and in reclaim action we should just check current queue's capability, and reclaim happens when queue not allocatable so we just need focus on the reclaim here, so it's more descriptive to user preempt related semantics.